src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -3096,15 +3096,19 @@ namespace FanCoilUnits {
                                                    PLR,
                                                    CompressorOnFlag);
                     }
-                }
-                Calc4PipeFanCoil(state, FanCoilNum, ControlledZoneNum, FirstHVACIteration, QUnitOut);
+                } else if ((CoolingLoad && QUnitOutMax > QZnReq && QZnReq < 0.0) || (HeatingLoad && QUnitOutMax < QZnReq && QZnReq > 0.0)) {
+                    // load is larger than capacity, thus run the fancoil unit at full capacity
+                    PLR = 1.0;
+                } 
+                Calc4PipeFanCoil(state, FanCoilNum, ControlledZoneNum, FirstHVACIteration, QUnitOut, PLR);
                 PowerMet = QUnitOut;
                 AirMassFlow = Node(InletNode).MassFlowRate;
                 // CR9155 Remove specific humidity calculations
                 SpecHumOut = Node(OutletNode).HumRat;
                 SpecHumIn = Node(InletNode).HumRat;
                 // Latent rate (kg/s), dehumid = negative
                 LatOutputProvided = AirMassFlow * (SpecHumOut - SpecHumIn);
+                FanCoil(FanCoilNum).PLR = PLR;
```

Comment:
Needed to set it here for unit test.

Reply:
This is the one thing that stands out to me from your walkthrough.  Fixing it for the unit test just feels odd.  Can you clarify what impact this has on the simulation itself?

Reply:
PLR was not reported/updated for ASHRAE90VariableFan capacity control method. If you look at other capacity control methods PLR value is set to FanCoil.PLR for reporting purpose. In this fix I'm passing PLR as argument to Calc4PipeFanCoil() function [here](https://github.com/NREL/EnergyPlus/blob/173380574_Issue8087/src/EnergyPlus/FanCoilUnits.cc#L3103) and setting PLR value to 1.0 when the load exceeds the capacity [here](https://github.com/NREL/EnergyPlus/blob/173380574_Issue8087/src/EnergyPlus/FanCoilUnits.cc#L3099-L3102). There is not other place where a value is set to FanCoil.PLR for this capacity control method.

Reply:
OK, I'll accept that, sounds good.